src/core/geometry/qgspoint.cpp

```diff
@@ -306,6 +306,11 @@ json QgsPoint::asJsonObject( int precision ) const
   return j;
 }
 
+QString QgsPoint::asKML( int precision ) const
+{
+  return QStringLiteral( "<Point><coordinates>%1,%2</coordinates></Point>" ).arg( qgsDoubleToString( mX, precision ) ).arg( qgsDoubleToString( mY, precision ) );
```

Comment:
Did the proposed single-arg version not work here? This method will be called often and should perform well.

Reply:
No was just me who missed it - sorry.